Author: Steffen Moeller <moeller@debian.org>
Description: Adjusted path for BOINC to server for python code
Index: boinc/py/boinc_path_config.py.in
===================================================================
--- boinc.orig/py/boinc_path_config.py.in
+++ boinc/py/boinc_path_config.py.in
@@ -8,5 +8,5 @@
 
 import sys, os
 
-TOP_BUILD_DIR = "@TOP_BUILD_DIR@"
+TOP_BUILD_DIR = "/usr/share/boinc-server-maker"
 sys.path.insert(0, os.path.join(TOP_BUILD_DIR, 'py'))
